__model__/api.model.js

```diff
@@ -0,0 +1,9 @@
+const db = require("../db/connection");
+const fs = require("fs/promises");
+
+exports.selectApi = () => {
+  return fs.readFile(`${__dirname}/../endpoints.json`, "utf-8").then((data) => {
+    const parsedData = JSON.parse(data);
+    return parsedData;
+  });
+};
```

Comment:
This is not interacting with the database so isn't strictly speaking a model, consider making this a util function if you want the logic separate from the controller